AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -1407,6 +1407,8 @@ combi.de#$#.modal--cookie-notice { display: none !important; }
 raeer.com###RaeerGoogleCookiesBox
 fashionunited.de##.MuiSnackbar-anchorOriginBottomLeft
 stoff4you.de###dxCookieConsentContainer
+# https://github.com/AdguardTeam/AdguardFilters/issues/114783
+onlinefinanzierung.vr-smart-finanz.de##.loading-transparent.loading.cookieconsentContainer
```

Comment:
`onlinefinanzierung.vr-smart-finanz.de##.cookieconsentContainer`is enough

Reply:
I have also moved the rule without comment. Please see how it should be - it will be helpful for your further commits.